docs/content/getting-started/signmykey/index.md

```diff
@@ -183,4 +183,38 @@ Look at Principals in the output:
         Critical Options: (none)
         Extensions: 
                 permit-pty
-```
\ No newline at end of file
+```
+
+## Additionnal infos
+
+### Deprecated algorithm
+
+In recent OS (Ubuntu 22.04), not yet on Redhat family (9) you can no longer ssh to servers with that OS because the RSA algorithm is deprecated.
```

Comment:
As discussed in #138, RSA is not deprecated, it's SHA-1 signatures that are deprecated. RSA keys are signed with SHA-1 or SHA-2. SHA-2 signatures are ok, but there is a subtle bug between some versions of openssh servers and clients that doesn't correctly announce that SHA-2 signatures are supported by the remote server for RSA keys. So the workaround is to use a non-RSA keys.